Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -136,7 +137,13 @@ private void openChildFormDrink(Drink drink)
             //closeActiveForm();            //#commentedarea
             configureChildForm(drinkForm);
         }
-        private void configureChildForm(Form childForm)
+        private void openChildFormUserContribution(Drink drink, Bar bar)
+        {
+            Child_Forms.UserContribution userContribution = new UserContribution(drink, bar);
```

Comment:
Did you check in with adam about this?

Reply:
don't know why it shows that I deleted this, it still there in form1.cs